exonum-java-binding/core/rust/ejb-app/TUTORIAL.md

```diff
@@ -18,90 +19,81 @@ in the [user guide](https://exonum.com/doc/version/0.10/get-started/java-binding
 
 ### Step 1. Configure Environment
 
-`EJB_ROOT` used in examples of this section corresponds to the Java Binding root directory.
-
-#### `LD_LIBRARY_PATH` and `EJB_LIBPATH`
+#### `LD_LIBRARY_PATH`
 
 `LD_LIBRARY_PATH` is required to locate native libraries used by Java Binding.
-You need to provide paths to:
-  - JVM library (e.g., `libjvm.so` on Linux).
-  - Rust standard library that is used to build the application.
-  - Application libraries used by Java Binding.
+You need to provide path to JVM library (e.g. `libjvm.so` on Linux).
 
 You can use the following script for this purpose:
 
 ```bash
 JAVA_HOME="${JAVA_HOME:-$(java -XshowSettings:properties -version 2>&1 > /dev/null | grep 'java.home' | awk '{print $3}')}"
 LIBJVM_PATH="$(find ${JAVA_HOME} -type f -name libjvm.* | xargs -n1 dirname)"
 
-RUST_LIB_PATH="$(rustup run 1.32.0 rustc --print sysroot)/lib"
-
-export EJB_LIBPATH="${EJB_ROOT}/core/rust/target/debug"
-
-export LD_LIBRARY_PATH="${LIBJVM_PATH}:${RUST_LIB_PATH}:${EJB_LIBPATH}"
+export LD_LIBRARY_PATH="${LIBJVM_PATH}"
```

Comment:
let's remove LD_LIBRARY_PATH and export LIBJVM_PATH

Reply:
It's impossible, we need to export LD_LIBRARY_PATH